memfault/memfault-docs
# style(docs): MFLT-11822 - Update arrows styling

### Summary

To provide more consistent, semantically sound, and beautiful documentation, arrows have been updated from `->` or `>` to `→`, used mostly when referencing a menu hierarchy.  I also considered modifying our linter to replace them, but did not want to collide with other functional uses in comments or code examples, etc.

### Internal Documentation

Aligns with https://github.com/memfault/memfault/pull/11178

### Test Plan

1. Run docs locally
2. Observe changed pages
3. See glorious arrows
